apps/remix-ide-e2e/src/tests/solidityUnittests.test.ts

```diff
@@ -94,10 +93,9 @@ module.exports = {
       .waitForElementPresent('*[data-id="testTabRunTestsTabRunAction"]')
       .clickElementAtPosition('.singleTestLabel', 0)
       .scrollAndClick('*[data-id="testTabRunTestsTabRunAction"]')
-      .pause(2000)
       .click('*[data-id="testTabRunTestsTabStopAction"]')
-      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'tests/ks2b_test.sol', 200000)
```

Comment:
this test has been changed too and this is the other way around...

Reply:
I checked it locally. It is working like that. It may be because logic for showing files list is changed so now tests run for Ballot_test.sol and after clicking `Stop`, tests don't run for `ks2b_test.sol`